index.html

```diff
@@ -47,20 +49,8 @@ <h1>CODEVID-19</h1>
       <p>
         With the COVID-19 pandemic in full swing, many people are looking at a
         lot of free time stuck indoors. Avoid cabin fever and use that time to
-        make a difference in your community and the fight against COVID-19 by
-        joining CODEVID-19, the world’s first pandemic hackathon!
-      </p>
-      <p>
-        The purpose of the hackathon ultimately isn’t to win prizes, it’s to
-        build useful apps that people can use to manage and survive during the
-        COVID-19 pandemic. What started in Edmonton, Canada as a discussion
-        amongst developers on the Dev Edmonton Society Slack has exploded into a
-        global initiative to improve people's quality of live and fight the
-        pandemic.
-      </p>
-      <p>
-        With participants in over 10 countries around world, it's a growing
-        movement. Do your part and join the fight now!
+        make a difference in your community: fight against COVID-19 by joining
```

Comment:
I'd like to include the international reference here, biut agree this could be way shorter. What about:

> With the COVID-19 pandemic in full swing, many people are looking at a lot of free time stuck indoors. Avoid cabin fever and join participants from more than 10 countries by fighting COVID-19 as part of the world's first global pandemic hackathon.
>
> "Join The Fight!"